client/signup/steps/domains/index.jsx

```diff
@@ -121,14 +122,22 @@ class DomainsStep extends React.Component {
 		}
 
 		this.showTestCopy = false;
+		this.showDesignUpdate = false;
 
 		// Do not assign user to the test if either in the launch flow or in /start/{PLAN_SLUG} flow
 		if (
 			false !== this.props.shouldShowDomainTestCopy &&
```

Comment:
This is not part of the PR, but I'm curious if this is intentional since this expression will be true because `shouldShowDomainTestCopy` is `null`.

Reply:
Yep as the code comment above it says, the user is not to be assigned to the test in certain flows. So, if `shouldShowDomainTestCopy ` is null then we should proceed to assign the user to the test.